parts/k8s/kubernetesinstalls.sh

```diff
@@ -316,3 +317,13 @@ cleanUpContainerImages() {
         docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep 'nginx') &
     fi
 }
+
+overrideNetworkConfig() {
```

Comment:
this is the workaround suggested by the az linux team to workaround #761

Reply:
We don't want to do this via cloud-init?

Reply:
because there is a cloud init bug that is preventing this from being user overridable. @jasonzio from AzLinux has a PR to fix it open but it might take a while before it gets in. See the email thread for more context.

Reply:
thanks for the context!